docs/operator-config.asciidoc

```diff
@@ -0,0 +1,61 @@
+[id="{p}-operator-config"]
+== Operator configuration
+
+The operator exposes several configuration options. Unless otherwise noted, they can be set as environment variables or command line flags.
+
+
+[width="70%",valign="middle",halign="center",options="header"]
+|==========================
+|Flag |Type|Default|Description
+
+|log-verbosity |int |0 |Verbosity level of logs. -2=Error, -1=Warn, 0=Info, >0=Debug
+|enable-debug-logs |bool |false |Enables debug logs. Equivalent to `log-verbosity=1`
+|metrics-port |int |0 |Port to use for exposing metrics in the Prometheus format. Set 0 to disable
```

Comment:
Default metric port is 8080, it means that it is enabled by default: https://github.com/elastic/cloud-on-k8s/blob/8f7265b227a7e854edcb6dec46c47569a26350bf/cmd/manager/main.go#L48

Reply:
Ah good catch